src/Core/maps/src/Platform/iOS/MauiMKMapView.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+using System.Collections;
+using System.Linq;
+using CoreLocation;
+using MapKit;
+using Microsoft.Maui.Maps.Handlers;
+using Microsoft.Maui.Platform;
+using ObjCRuntime;
+using UIKit;
+
+namespace Microsoft.Maui.Maps.Platform
+{
+	internal class MauiMKMapView : MKMapView
```

Comment:
Is it fine to be inteernal ?

Reply:
No, this needs to be public so that users that interact with our MAUI factory can use `MauiMKMapView` 

I don't really get why we want to use the least derived type on the handler generics either.  The iOS `MapHandler` flat out won't work if the user uses a `MKMapView` right? 

If there's a valid use case for a MapHandler that uses `MKMapView` vs `MAuiMKMapView` then ok but I don't think there is.

Reply:
Ok moved to MauiMKMapView